src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,276 @@
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Card } from "../../Components/Cards/Cards";
+import { Filter } from "../../Components/Filter/Filter";
```

Comment:
suggestion: you can create a single file for exporting components and import all ur components from that file